### PR TITLE
Fix 'Cannot assign requested address' issue at travis

### PR DESCRIPTION
**Pull request by phoad**
Suggested way of fixing the issue:
 - https://github.com/travis-ci/travis-ci/issues/8711

### CODE REVIEW

**Comment by phoad** [comment]
It did not work :(
@vitaut -- can you help us about this. Do you have expertise on Travis?

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Travis CI config for rsocket-cpp.
 
-sudo: false
+sudo: true
```
Comment (vitaut) [inline .travis.yml]: Please don't do this, because this will switch back to old slow infra.

**Comment by vitaut** [comment]
I'd wait to hear what Travis folks say, the issue has just been opened. Or try using IPv4.

**Comment by somasun** [comment]
Right, I remember reverting to IPv4 last time I was seeing this issue.

**Comment by phoad** [comment]
Abandon, already fixed by @somasun

**Pull request closed without merging**